README.md

```diff
@@ -145,7 +145,7 @@ npx react-codemod sort-comp <path>
 
 #### `update-react-imports`
 
-[As of Babel 7.9.0](https://babeljs.io/blog/2020/03/16/7.9.0#a-new-jsx-transform-11154-https-githubcom-babel-babel-pull-11154), when using `runtime: automatic` in `@babel/preset-react` or `@babel/plugin-transform-react-jsx`, you will not need to explicitly import React for compiling jsx. This codemod removes the redundant import statements. It also converts  (`import React from 'react'`) to named imports (`import * as React from 'react`).
+[As of Babel 7.9.0](https://babeljs.io/blog/2020/03/16/7.9.0#a-new-jsx-transform-11154-https-githubcom-babel-babel-pull-11154), when using `runtime: automatic` in `@babel/preset-react` or `@babel/plugin-transform-react-jsx`, you will not need to explicitly import React for compiling jsx. This codemod removes the redundant import statements. It also converts  (`import React from 'react'`) to destructured named imports (`import { useState } as React from 'react`).
```

Comment:
This edit seems a bit oddly structured, might want to edit in (it's fine if it's in a future PR, not critical). Might not even need it, tbh.

Reply:
kay removed this edit for now